calls/v1/trials/TripalWebServiceBrapiV1Trials.inc

```diff
@@ -233,7 +235,7 @@ class TripalWebServiceBrapiV1Trials extends TripalWebServiceDatabaseCall {
           }
         }
 
-        $project->studies = [ $all_studies ];
+        $project->studies = (array) ['set' => $all_studies];
```

Comment:
This line makes no sense... Casting an associative array as an array will accomplish nothing... instead $all_studies should be an array of objects and used directly.